resources/css/bem/input-container.less

```diff
@@ -50,6 +50,11 @@
     }
   }
 
+  &--beatmap-owner-editor {
```

Comment:
block interleaving aside, this seems wrong...? barely any way to differentiate editing and non-editing state. and clicking on the highlighted parts focuses somewhere unrelated if there's an edit state somewhere. or if there's nothing being edited, it just does nothing which is also weird despite having a hover outline.

I think the "beatmap owner" block can still use a background here... the &times; button can still look like belong to either to its left or right.

and looking again, maybe just remove the non-editing state and have them all editable right away.

![](https://s.kohaku.love/2024/11/firefox_2024-11-08_22-29-37.png)